tests/e2e/taqueria-plugin-flextesa.spec.mts

```diff
@@ -176,6 +177,11 @@ describe('E2E Testing for taqueria flextesa plugin sandbox starts/stops', () =>
 		}
 	});
 
+	test.only('Verify that Taqueria accepts any origin and does not emit any CORS related errors', async () => {
```

Comment:
This only should be removed

Reply:
Ah, thanks - it snuck in there while debugging the node-fetch issue I mentioned.